Language/fa/Security.php

```diff
@@ -12,6 +12,7 @@
 // Security language settings
 return [
     'disallowedAction' => 'عملیات درخواستی شما مجاز نیست.',
+    'insecureCookie'   => 'تلاشی برای ارسال یک کوکی ایمن از طریق یک اتصال ناامن انجام شده',
```

Comment:
Please use the DOT(.) at the end of the translation.

Related to https://github.com/codeigniter4/CodeIgniter4/pull/8818